.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -0,0 +1,43 @@
+# .NET Core Release Checklist
+
+## Release Versions
+
+_The set of .NET Core versions that are being released as a unit._
+
+* 2.1.X: <runtime/SDK>
+* 3.1.X: <runtime/SDK>
+
+
+
+1. - [ ] Incase of 2.x, run auto-update(refer docs). Incase of 3.X:
```

Comment:
Break down into 2.x vs. 3.x more directly?

```markdown
1. [ ] Update target branch to new version.
   - For 2.x:
     - [ ] If releasing 2.x, run auto-update (refer to docs).
   - For 3.x:
     - [ ] if maestro auto-PR is active, then verify SHA1s in version.Details with manifest versions in VSU share dir (refer docs), make sure that we match the versions
     - [ ] in-case auto-PR updates are unavailable, checkout a local branch for the pertinent version and run darc updates(refer docs). 
       - [ ] Push this local branch upstream and start the release PR for the N version that is to be released.
```

1. [ ] Update target branch to new version.
   - For 2.x:
     - [ ] If releasing 2.x, run auto-update (refer to docs).
   - For 3.x:
     - [ ] if maestro auto-PR is active, then verify SHA1s in version.Details with manifest versions in VSU share dir (refer docs), make sure that we match the versions
     - [ ] in-case auto-PR updates are unavailable, checkout a local branch for the pertinent version and run darc updates(refer docs). 
       - [ ] Push this local branch upstream and start the release PR for the N version that is to be released.